manifests/package.pp

```diff
@@ -45,14 +45,20 @@
       $repo_require = undef
 
       $pkg_version = inline_template("<%= scope.lookupvar('sensu::version').sub(/(.*)\\-/, '\\1.') %>")
-      $pkg_title = 'Sensu'
-      $pkg_name = 'Sensu'
-      $pkg_source = "C:\\Windows\\Temp\\sensu-${sensu::version}.msi"
-      $pkg_require = "Remote_file[${pkg_source}]"
+      # Download the latest published version
+      $pkg_url_version = $::sensu::version ? {
+        installed => 'latest',
+        default   => $pkg_version,
+      }
```

Comment:
The specific links will be based on the OS architecture and Major release.  For example, the default computed links look like:

 * `https://repositories.sensuapp.org/msi/2012r2/sensu-latest-x86.msi`
 * `https://repositories.sensuapp.org/msi/2012r2/sensu-latest-x64.msi`
 * `https://repositories.sensuapp.org/msi/2012/sensu-latest-x86.msi`
 * `https://repositories.sensuapp.org/msi/2012/sensu-latest-x64.msi`
 * `https://repositories.sensuapp.org/msi/2008r2/sensu-latest-x86.msi`
 * `https://repositories.sensuapp.org/msi/2008r2/sensu-latest-x64.msi`